packages/commercetools/theme/pages/index.vue

```diff
@@ -155,12 +155,23 @@ import {
   SfImage,
   SfBannerGrid
 } from "@storefront-ui/vue";
-import { setup, getProduct } from '@vue-storefront/commercetools-api'
+import { getProduct } from '@vue-storefront/commercetools-api'
+
+function encodeURIAll(text) {
+  return encodeURIComponent(text).replace(/[!'()~*]/g, (c) => {
+    return '%' + c.charCodeAt(0).toString(16);
+  });
+}
 
 export default {
   name: "Home",
   transition: 'fade',
   data() {
+    getProduct({ variables: {
```

Comment:
@filrak example how to use it, I'll implement a generic helpers (eg. `getWhereByCatId`, `getWhereByCatSlug` etc.) to generate `where` param